lib/MentionsInput.js

```diff
@@ -445,10 +446,10 @@ module.exports = React.createClass({
   updateSuggestionsPosition: function() {
     if(!this.refs.caret || !this.refs.suggestions) return;
 
-    var containerEl = this.refs.container.getDOMNode();
-    var caretEl = this.refs.caret.getDOMNode();
-    var suggestionsEl = this.refs.suggestions.getDOMNode();
-    var highligherEl = this.refs.highlighter.getDOMNode();
+    var containerEl = this.refs.container;
+    var caretEl = this.refs.caret;
+    var suggestionsEl = ReactDOM.findDOMNode(this.refs.suggestions);
```

Comment:
Yes in all above examples you can, but here it refers to separate React Component, you can try yourself :)